test/use-swr-config-callbacks.test.tsx

```diff
@@ -88,13 +88,13 @@ describe('useSWR - config callbacks', () => {
     // props changed, but the onError callback doese not trigger yet.
     rerender(<Page text="b" />)
     expect(container.firstChild.textContent).toMatchInlineSnapshot(`"Error: 0"`)
-    expect(container.firstElementChild.getAttribute('title')).toEqual('b')
```

Comment:
This raises an ESLint error and has been fixed automatically by the ESLint plugin.
In #933, this would be replaced with the way that doesn't use `container`, so it's a temporary solution.